CHANGES.md

```diff
@@ -1,6 +1,11 @@
 # Change Log
 
 ## Unreleased
+- Dependencies: Update to sphinx-design 0.6.0
+  See also [sphinx-design change log].
+- Remove support for Python 3.7 and 3.8, following `sphinx-design`.
```

Comment:
That's a drawback. Some projects at https://github.com/crate will probably still need Python 3.8.